driver/ppm_fillers.c

```diff
@@ -243,8 +244,8 @@ const struct ppm_event_entry g_ppm_events[PPM_EVENT_MAX] = {
 	[PPME_SYSCALL_LINKAT_X] = {f_sys_single_x},
 	[PPME_SYSCALL_UNLINK_E] = {f_sys_single},
 	[PPME_SYSCALL_UNLINK_X] = {f_sys_single_x},
-	[PPME_SYSCALL_UNLINKAT_E] = {PPM_AUTOFILL, 2, APT_REG, {{0}, {1} } },
-	[PPME_SYSCALL_UNLINKAT_X] = {f_sys_single_x},
+	[PPME_SYSCALL_UNLINKAT_3_E] = {f_sys_unlinkat_e},
```

Comment:
Remind me why PPM_AUTOFILL doesn't work here?

Reply:
Can `PPM_AUTOFILL` handle syscall flags?
I have a call to convert the flags to `PPM_` flags:
`static inline u32 unlinkat_flags_to_scap(unsigned long flags)`
(From the [documentation](https://linux.die.net/man/2/unlinkat) there is only one flag supported `AT_REMOVEDIR`)